docker/Dockerfile-test-install.ubuntu.xenial

```diff
@@ -23,4 +18,9 @@ RUN apt-get update && apt-get install -y --no-install-suggests --no-install-reco
   python3 \
   zip
 
+RUN apt install -y software-properties-common
+RUN add-apt-repository ppa:mhier/libboost-latest
+RUN apt update
+RUN apt install -y libboost1.67
```

Comment:
Why is this needed for test-install? We install the package built against stock libboost in this case.

Reply:
Boost shared library must be installed on system where we run aktualizr, this is installation from alternative repository which has version of the boost 1.67, the default repo for xenial allows to install only 1.58

Reply:
Yes but that's what gets compiled on the actual debian package and we probably want to keep it like that. It doesn't make sense to provide a xenial deb package and ask people to install a crazy new boost version.

Reply:
Here we back to the question which is the minimum boost library we should use! 

From boost FAQ
How can the Boost libraries be used successfully for important projects? Many of the Boost libraries are actively maintained and improved, so backward compatibility with prior version isn't always possible. Deal with this by freezing the version of the Boost libraries used by your project. Only upgrade at points in your project's life cycle where a bit of change will not cause problems. Individual bug fixes can always be obtained from the boost repository.

https://www.boost.org/users/faq.html

Reply:
Ah yes I just realized that we also build the xenial package against this new boost then... That makes this .deb package quite useless. It doesn't make sense to say we support xenial if we recompile most of our dependencies there.
And we don't even need a new boost for the application, only the tests!